hs_discover/package.json

```diff
@@ -14,7 +14,10 @@
     "axios": "^1.6.0",
     "bootstrap": "^4.4.1",
     "bootstrap-vue": "^2.14.0",
+    "braces": "^3.0.3",
     "core-js": "^3.8.3",
+    "express": "^4.19.2",
+    "follow-redirects": "^1.15.6",
```

Comment:
@devincowan - I searched the project and I see no usage of this `follow-redirects` package. Is it going to be used?

Reply:
it gets installed as a dependency of 
axios
https://github.com/hydroshare/hydroshare/blob/master/hs_discover/package-lock.json#L2699
and http-proxy
https://github.com/hydroshare/hydroshare/blob/master/hs_discover/package-lock.json#L5408

we needed to ensure > 1.15.6
https://github.com/hydroshare/hydroshare/security/dependabot/62
https://github.com/hydroshare/hydroshare/security/dependabot/61

Reply:
I tried to have `npm audit fix` update the versions directly in the package-lock.json but it didn't work for some reason